force-app/main/default/classes/LeadConversion.cls

```diff
@@ -0,0 +1,68 @@
+public without sharing class LeadConversion {
+  @InvocableMethod(Label='Convert Leads')
+  public static void convertLeads(List<LeadConversionRequest> requests) {
+    // Get leads to convert
+    List<Lead> leadsToConvert = [SELECT Id, IsConverted FROM Lead];
```

Comment:
Will want to filter this SOQL query by the Lead IDs in the list of requests that came into the method. 

```java
Set<ID> leadIds = new Set<ID>();
for (LeadConversionRequest lcr : requests) {
    leadIds.add(lcr.leadId);
}

List<Lead> leadsToConvert = [
    SELECT Id, IsConverted 
    FROM Lead 
    WHERE Id IN :leadIds 
        AND IsConverted = False
];
```